sormas-backend/src/main/java/de/symeda/sormas/backend/task/TaskFacadeEjb.java

```diff
@@ -157,6 +157,9 @@ public Task fromDto(TaskDto source, boolean checkChangeDate) {
 		target.setAssigneeReply(source.getAssigneeReply());
 		target.setCreatorUser(userService.getByReferenceDto(source.getCreatorUser()));
 		target.setCreatorComment(source.getCreatorComment());
+		if (source.getObserverUsers() != null) {
+			target.setObserverUsers(source.getObserverUsers().stream().map(userService::getByReferenceDto).collect(Collectors.toList()));
```

Comment:
Please add a new method called `getByReferenceDtos` to `BaseAdoService` that can be passed a list of ReferenceDtos and that returns a list of ADOs, and use this method here instead of mapping all the observer users individually (because that will lead to more individual database operations).